src/components/task-plan/homework/exercises.cjsx

```diff
@@ -184,6 +185,9 @@ ExerciseTable = React.createClass
     {section, lo, tagString} = ExerciseStore.getTagStrings(exerciseId)
     content = ExerciseStore.getContent(exerciseId)
 
+    if $("<div>#{content}</div>").children().length > 1
```

Comment:
Can you use `document.createElement('div').innerHTML = content` here for the test? instead of including jQuery